nautobot/docs/user-guide/administration/installation/index.md

```diff
@@ -18,7 +21,79 @@ For more information about the Docker tags, Docker configurations, or using cont
 
     Nautobot *should* work on any POSIX-compliant system including practically any flavor of Linux, BSD, or even macOS, but those are not *officially* supported at this time.
 
-To begin installing Nautobot, click the link to your preferred deployment approach from the list above.
+
+#### Bare metal / Virtual Machine Pros
+
+- Familiar deployment option for many administrators
+- Performance is generally higher without the overhead of virtualization
+- Full control over hardware and software allows for greater customization and security practices
+
+#### Bare metal / Virtual Machine Cons
+
+- Maintenance and high availability is more difficult than Docker or Cloud options
+- Preparing server for production grade security and scaling requires additional effort 
+
+### Docker Compose
+
+Nautobot can be deployed via Docker using [Nautobot Docker images](../guides/docker.md) that are available either:
+
+* [Docker Hub](https://hub.docker.com/r/networktocode/nautobot)
+* [GitHub Container Registry](https://github.com/nautobot/nautobot/pkgs/container/nautobot) 
+
+For local testing and lab work you can use the Nautobot [Docker Compose](https://github.com/nautobot/nautobot-docker-compose) project which leverages the container images above. 
+
+For more information about the Docker tags, Docker configurations, or using container images for your own development environment, see [Nautobot in Docker](../guides/docker.md).
+
+#### Docker Compose Pros
+
+- Quick to deploy for single node lab work and testing
+- For those looking for a multi-container single-node install for Nautobot often coupled with backup & HA capabilities from their hypervisor manager
+
+#### Docker Compose Cons
+
+- Does not deploy NGINX, SSL, or any Nautobot plugins by default
```

Comment:
I don't *think* this entirely accurate, though I see a similar message in the nautobot-docker-compose README.md - the baseline Docker images do include SSL with a self-signed certificate by default

The nautobot-docker-compose repo also provides documentation for adding SSL certificates and/or adding Nautobot ~plugins~ *Apps* to the image.

Reply:
Gotcha, I will just take that line out then. Thanks!